src/azul/service/hca_response_v5.py

```diff
@@ -287,14 +289,61 @@ def agg_values(function: Callable[[JSON], T], *path: str) -> List[T]:
             totalFileSize=agg_value('totalFileSize', 'value'),
             donorCount=agg_value('donorCount', 'value'),
             labCount=agg_value('labCount', 'value'),
+            # FIXME: Remove deprecated fields totalCellCount and projectEstimatedCellCount
+            #        https://github.com/DataBiosphere/azul/issues/3650
             totalCellCount=agg_value('totalCellCount', 'value'),
             projectEstimatedCellCount=agg_value('projectEstimatedCellCount', 'value'),
```

Comment:
We should be able to derive these from the new representation and eliminate the actual aggregation in the ES request, thereby improving performance.